src/app/services/[id]/databases/page.tsx

```
@@ -0,0 +1,36 @@
+const ServiceDatabasesPage = async ({
+  params,
+}: ServiceDatabasesPageProps) => {
+  const service = await getService(db, params.id);
+  if (!service) notFound();
+  return (
+    <PageSection
+      title={`Databases of ${service.name}`}
+      toolbar={(
+        <>
+          <BackButton fallback={`/services/${params.id}`} />
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)